app/views/compute_resources_vms/form/ovirt/_volume.html.erb

```diff
@@ -8,7 +8,7 @@
 <%= checkbox_f f, :preallocate, {:disabled => !new_vm, :checked => f.object.sparse == 'false', :help_inline => _('Uses thin provisioning if unchecked'), :label => _('Preallocate disk'), :label_size => "col-md-2" } %>
 <%= checkbox_f f, :wipe_after_delete, {:disabled => !new_vm, :checked => f.object.wipe_after_delete  == 'true', :label => _('Wipe disk after delete'), :label_size => "col-md-2" } %>
 <%= select_f f, :interface, [OpenStruct.new({:id =>'virtio', :name =>'VirtIO'}), OpenStruct.new({:id =>'virtio_scsi', :name =>'VirtIO SCSI'}), OpenStruct.new({:id =>'ide', :name =>'IDE'})],
-             :id, :name, { :include_blank => true }, :label => _('Disk interface'), :label_size => "col-md-2", :disabled => !new_vm, :class => "col-md-2" %>
+  :id, :name, { :selected => 'virtio_scsi' }, :label => _('Disk interface'), :label_size => "col-md-2", :disabled => !new_vm, :class => "col-md-2" %>
```

Comment:
Isn't this going to select `virtio_scsi` even if the value is set already as well? :(
Suboptimal solution would be to put this at the beggining of the template.
```erb
<% f.object['virtio_scsi'] ||= 'virtio_scsi' if f.object %>
```

Reply:
Maybe reordering options is sufficient, as virtio_scsi is the default for oVirt for some version, we can put it at the to, so we don't need to use `selected` as it will be selected by default if no value is present.

Reply:
Yeah, that's much better solution :dart: :+1: